tests/test_exceptions.py

```diff
@@ -140,3 +140,18 @@ def test_format_sacred_error(print_traceback, filter_traceback, print_usage, exp
     except SacredError as e:
         st = format_sacred_error(e, "usage")
         assert re.match(expected, st, re.MULTILINE)
+
+
+def test_chained_error():
+    try:
+        try:
+            print(1 / 0)
+        except Exception as e:
+            raise SacredError("Something bad happened") from e
+    except SacredError as e:
+        st = format_sacred_error(e, "usage")
+        assert re.match(
+            r"Traceback \(most recent calls WITHOUT Sacred internals\):\n  File \"[^\"]+?test_exceptions.py\", line 148, in test_chained_error\n    print\(1 / 0\)\nZeroDivisionError: division by zero\n\nThe above exception was the direct cause of the following exception:\n\nTraceback \(most recent calls WITHOUT Sacred internals\):\n  File \"[^\"]+?test_exceptions.py\", line 150, in test_chained_error\n    raise SacredError\(\"Something bad happened\"\) from e\nsacred.utils.SacredError: Something bad happened\n",
```

Comment:
I don't like the hard-coded line numbers here. These would break this testecase every time something earlier in this file is modified. How about matching against intgers? I also find this quite hard to read, how about the following formatting?

```python
assert re.match(
            r"Traceback \(most recent calls WITHOUT Sacred internals\):\n"
             "  File \"[^\"]+?test_exceptions.py\", line \d+, in test_chained_error\n"
             "    print\(1 / 0\)\nZeroDivisionError: division by zero\n\n"
             "The above exception was the direct cause of the following exception:\n\n"
             "Traceback \(most recent calls WITHOUT Sacred internals\):\n"
             "  File \"[^\"]+?test_exceptions.py\", line \d+, in test_chained_error\n"
             "    raise SacredError\(\"Something bad happened\"\) from e\nsacred.utils.SacredError: Something bad happened\n",
```

Reply:
Done